.github/workflows/testing.yml

```diff
@@ -1,9 +1,6 @@
 name: "Run Tests"
 
 on:
-  push:
-    branches:
-      - "main"
```

Comment:
Is this a change you want merged?

Reply:
I deleted these lines to skip the test when the CHANGELOG is updated (there is a push to main). Do you think this is the best way to do that? I believe we don't usually have direct commits pushes to main, and the Docs build is not affected by skipping those tests.

Reply:
I'm thinking there might be a way to filter based on tags or the commit message/PR title, so we could run it generally, but skip commits from the changelog Action.